src/main/java/in/twizmwaz/cardinal/module/modules/startTimer/StartTimer.java

```diff
@@ -74,6 +89,18 @@ public void run() {
                     }
                 }
             }
+            for (Player player : Bukkit.getOnlinePlayers()) {
+                if (!TeamUtils.getTeamByPlayer(player).isObserver()) {
+                    if (time == 0) {
+                        //sendTitle(ChatColor.GREEN + "Go!");
+                    } else if (((time % 20) == 0) && (time <= 60)) {
+                        //sendTitle(ChatColor.YELLOW + (time / 20));
+                    }
```

Comment:
Already tested this sending messages to the chat, this is what I got. http://gyazo.com/c2f9579f5a523fb005533810169a398a